tests/ci/clean_up.sh

```diff
@@ -11,13 +11,14 @@ ID=${ID:-}
 IMAGE_KEY=${IMAGE_KEY:-}
 TEMPDIR=${TEMPDIR:-}
 HTTPD_PATH=${HTTPD_PATH:-}
+DOWNLOAD_DIRECTORY="/var/lib/libvirt/images"
 
 greenprint "🧼 Cleaning up: start"
 # Remove stop and remove the VM
 virsh destroy "${IMAGE_KEY}" || true
 virsh undefine "${IMAGE_KEY}" --nvram
```

Comment:
@guy9050 I wonder if this will fail if the new script (`sync-to-aws.sh`) will successfully remove the VM.
Maybe you could add `|| true`, as in the line above, just in case, so the script continue if the VM has been already removed.

Reply:
Indeed it happened. The previous script `undefined` correctly the VM and when this script tried again, failed and exit with 1. So it never got to the `virsh vol-delete`.
Please, add her as well the `|| true`.
Thanks.

Reply:
Thanks! done